octavia_f5/cmd/house_keeping.py

```diff
@@ -0,0 +1,78 @@
+# Copyright 2015 Hewlett-Packard Development Company, L.P.
```

Comment:
Maybe add SAP copyright?

Reply:
Oh... I'll add it to status_manager too, since it's missing there as well.